Pcap++/header/PcapLiveDevice.h

```diff
@@ -80,16 +80,30 @@ namespace pcpp
 		friend class PcapLiveDeviceList;
 
 	protected:
+		/// @struct DeviceInterfaceDetails
+		/// @brief A struct that contains all details of a network interface.
+		struct DeviceInterfaceDetails
+		{
+			explicit DeviceInterfaceDetails(pcap_if_t* pInterface);
+			/// @brief Name of the device.
+			std::string name;
+			/// @brief Description of the device.
+			std::string description;
+			/// @brief IP addresses associated with the device.
+			std::vector<IPAddress> addresses;
+			/// @brief Flag to indicate if the device is a loopback device.
+			bool isLoopback;
+		};
```

Comment:
very nit: no need for doxygen documentation for private/protected structs

Reply:
Sure, its not strictly required, but I think its fine to stay, right?

Reply:
Sure we can leave it 🙂